qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -256,10 +264,11 @@ def __init__(self, name: str, address: str, visalib=None,
                                           y=self.GRPY.field(),
                                           z=self.GRPZ.field())
 
-        for coord in ['x', 'y', 'z', 'r', 'theta', 'phi', 'rho']:
+        for coord, unit in zip(['x', 'y', 'z', 'r', 'theta', 'phi', 'rho'],
+                               ['T', 'T', 'T', 'T', 'degrees', 'degres', 'T']):
```

Comment:
typos. Also, what is SI for degrees? `deg`/`degrees`/else?
```suggestion
                               ['T', 'T', 'T', 'T', 'deg', 'deg', 'T']):
```

Reply:
Degrees is not an SI unit, although it is acceptable to use. The "correct" symbol is °, but I would vote for just spelling out "degrees".

Sources: https://www.bipm.org/en/publications/si-brochure/table6.html

Reply:
```suggestion
                               ['T', 'T', 'T', 'T', 'degrees', 'degrees', 'T']):
```